website/docs/reference/resource-properties/anchors.md

```diff
@@ -0,0 +1,136 @@
+
+<File name='models/_models.yml'>
+
+```yml
+anchors: 
+  - &id_column_alias
+      name: id
+      description: This is a unique identifier.
+      data_type: int
+      data_tests:
+        - not_null
+        - unique
+
+models:
+  - name: my_first_model
+    columns: 
+      - *id_column_alias
+      - name: unrelated_column_a
+        description: This column is not repeated in other models.
+      - name: unrelated_column_b
+  - name: my_second_model
+    columns: 
+      - *id_column_alias
+      - name: unrelated_column_c
+  - name: my_third_model
+    columns: 
+      - *id_column_alias
+      - name: unrelated_column_d
+```
+
+</File>
+
+<Lightbox src="/img/reference/resource-properties/anchor_example_expansion.png" title="Behind the scenes, the alias is replaced with the object defined by the anchor."/>
+
+### Merge syntax
+
+Sometimes, an anchor is mostly the same but one part needs to be overridden. When the anchor refers to a dictionary/mapping (not a list or a <Term id="scalar-value" />), you can use the `<<:` merge syntax to override an already-defined key, or add extra keys to the dictionary. For example:
+
+<File name='models/_models.yml'>
+
+```yml
+anchors: 
+  - &id_column_alias
+      name: id
+      description: This is a unique identifier.
+      data_type: int
+      data_tests:
+        - not_null
+        - unique
+  - &source_template_alias
+    database: RAW
+    loader: fivetran
+    config:
+      freshness:
+        warn_after: {count: 1, period: day}
+
+models:
+  - name: my_first_model
+    columns: 
+      - *id_column_alias # brings in the full anchor defined above
+      - name: unrelated_column_a
+        description: This column is not repeated in other models.
+      - name: unrelated_column_b
+  - name: my_second_model
+    columns: 
+      - <<: *id_column_alias
+        data_type: bigint # overrides the data_type from int to bigint, while inheriting the name, description, and data tests
+      - name: unrelated_column_c
+  - name: my_third_model
+    columns: 
+      - <<: *id_column_alias
+        config:
+          meta: 
+            extra_key: extra_value # adds config.meta.extra_key to just this version of the id column, in addition to the name, description, data type, and data tests
+      - name: unrelated_column_d
+
+sources:
+  # both sources start with their database, loader, and freshness expectations set from the anchor, and merge in additional keys
+  - <<: *source_template_alias
+    name: salesforce
+    schema: etl_salesforce_schema
+    tables:
+      - name: opportunities
+      - name: users
+  - <<: *source_template_alias
+    name: hubspot
+    schema: etl_hubspot_schema
+    tables:
+      - name: contacts
+```
+
+</File>
+
+## Tips and common pitfalls
+
+<VersionBlock lastVersion="1.9">
+- Old versions of dbt Core (v1.9 and earlier) do not have a dedicated `anchors:` key. If you need to define a standalone anchor, you can leave it at the top level of your file.
```

Comment:
I wonder if this should be left unversioned? Seems like important information we'd want everyone to see.